scvi/models/totalvi.py

```diff
@@ -87,21 +91,22 @@ def __init__(
         self.n_input_proteins = n_input_proteins
         self.protein_dispersion = protein_dispersion
         self.latent_distribution = latent_distribution
+        self.protein_batch_mask = protein_batch_mask
 
         # parameters for prior on rate_back (background protein mean)
         if n_batch > 0:
             self.background_pro_alpha = torch.nn.Parameter(
                 torch.randn(n_input_proteins, n_batch)
             )
             self.background_pro_log_beta = torch.nn.Parameter(
-                torch.randn(n_input_proteins, n_batch)
+                torch.clamp(torch.randn(n_input_proteins, n_batch), -10, 1)
```

Comment:
if you need to clamp at the beginning, is it just because you did not initialize well ? There are some trunctated distributions to sample from for initialization

Reply:
It's not critical and is set to this in all the benchmarking code, so I think it's fine to leave for now?

Reply:
ok